snuba/query/processors/timeseries_column_processor.py

```diff
@@ -22,17 +22,17 @@ def time_expr(
             3600: FunctionCall(
                 alias,
                 "toStartOfHour",
-                (Column(None, column_name, None), Literal(None, "Universal")),
+                (Column(None, None, column_name), Literal(None, "Universal")),
```

Comment:
One of the most glorious usage of codemod I ever tried follows.
(Most of these changes were achieved with one since codemod call)